.github/workflows/test.yaml

```diff
@@ -115,9 +123,42 @@ jobs:
         test "$system_daily" == "$current_daily"
       shell: bash
 
+  required-static-checks:
```

Comment:
Since this is repeated twice, one option would be to make this into a composite action (like download-install-debian-deps) or a reusable workflow to eliminate code repetition and keep the test.yaml file short.

Reply:
Used composite action, let me know what you think